game/scripts/vscripts/internal/util.lua

```diff
@@ -106,8 +106,26 @@ function DebugAllCalls()
     end
 end
 
-
-
+--[[
+  Credits:
+    Angel Arena Blackstar
+  Description:
+    I don't really know what this does
```

Comment:
It looks like it returns the player id from a given unit / player / table. For example, you should be able to pass in a reference to a lycan wolf and get back the correct player's ID.

@Yahnich any insight on what this code does?